Assets/Scripts/Wall/WallSound.cs

```diff
@@ -0,0 +1,19 @@
+using UnityEngine;
+
+public class WallSound : MonoBehaviour
```

Comment:
Maybe it's easier to just implement this in `WallDanger`? It shouldn't complicated the class a lot - 1 serialized field, 1 constant and one call.